filebeat/prospector/prospector.go

```diff
@@ -111,7 +111,7 @@ func (p *Prospector) Run() {
 				}
 				select {
 				case <-p.done:
-					logp.Info("Prospector channel stopped")
+					logp.Debug("prospector", "Prospector channel stopped")
```

Comment:
I would probably keep these (all in this file) in Info as they show the shutdown messages (are only showed ones)